sdk/web-pubsub/arm-webpubsub/review/arm-webpubsub.api.md

```diff
@@ -531,7 +549,7 @@ export interface SystemData {
 
 // @public
 export interface TrackedResource extends Resource {
-    location?: string;
+    location: string;
```

Comment:
What is the swagger change for this breaking?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/22948/commits/385e57784bb8f103a8ef792f84c575a1c1652805#diff-f1b30594d5c33a24f727bebbe81bd04704cc186b09f8a6b0b575faba88d412adL2892